clang/lib/3C/ABounds.cpp

```diff
@@ -25,7 +25,7 @@ ABounds *ABounds::getBoundsInfo(AVarBoundsInfo *ABInfo, BoundsExpr *BExpr,
                                 const ASTContext &C) {
   ABounds *Ret = nullptr;
   CountBoundsExpr *CBE = dyn_cast<CountBoundsExpr>(BExpr->IgnoreParenCasts());
-  RangeBoundsExpr *RBE = dyn_cast<RangeBoundsExpr>(BExpr->IgnoreParenCasts());
+  //RangeBoundsExpr *RBE = dyn_cast<RangeBoundsExpr>(BExpr->IgnoreParenCasts());
```

Comment:
Unused variable I assume?

Reply:
Yes, this was generating a warning that we need to address.  The variable was used by the commented-out code below, so deleting it without deleting that code would leave the meaning of the other code unclear.  I thought commenting it out would be consistent.  Alternatively, I'd be happy to delete both pieces of code or use `#if 0` instead of comments (which is probably better practice in general).

Reply:
If it's used by commented code, then I agree that deleting it is a bad idea. If it can trivially be moved into the same block, I think that would be best.

Reply:
Done.  This seems like the obvious solution in hindsight; it's too bad I didn't think of it the first time.